components/alert/alert.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert Component</title>
+    <link rel="stylesheet" href="/library/component.css">
+    <style>
+        body {
+            padding: 0px;
+        }
```

Comment:
Internal CSS could be avoided you could make another file for this

Reply:
Following code is part of the support code for the CSS component library.
**Will be refactoring it once library P0s are complete.**